straxen/contexts.py

```diff
@@ -20,6 +20,7 @@
         straxen.PulseProcessing,
         straxen.Peaklets,
         straxen.PeakletClassification,
+        straxen.BayesPeakletClassification,
```

Comment:
you probably don't want to add this for 1T but for nT only in `xnt_common_opts`

Reply:
done